Source/BottomView/ImageStack.swift

```diff
@@ -0,0 +1,32 @@
+import UIKit
+
+public class ImageStack {
+
+  public struct Notifications {
+    public static let imageDidPushNotification = "imageDidPush:"
+    public static let imageDidDropNotification = "imageDidDrop:"
+    public static let stackDidReload = "stackDidReload:"
+    public static let imageKey = "image"
```

Comment:
It is `key` for `UIImage` in Dictionary. Decided to keep in notifications as it is the only place its being used.